cg/apps/scoutapi.py

```diff
@@ -66,8 +70,31 @@ def get_cases(self, case_id=None, institute=None, reruns=None, finished=None, ca
 
         else:
             models = self.cases(collaborator=institute, reruns=reruns,
-                               finished=finished, has_causatives=causatives,
-                               research_requested=research_requested,
-                               is_research=is_research, status=status)
+                                finished=finished, has_causatives=causatives,
+                                research_requested=research_requested,
+                                is_research=is_research, status=status)
 
         return models
+
+    def upload_delivery_report(self,
+                               report_path: str,
+                               case_id: str,
+                               update: bool = False):
+        """ Load a delivery report into a case in the database
+
+        If the report already exists the function will exit.
+        If the user want to load a report that is already in the database
+        'update' has to be 'True'
+
+        Args:
+            report_path (string):       Path to delivery report
+            case_id     (string):       Optional case identifier
```

Comment:
According to the function signature, it doesn't look optional ;)

Reply:
fixed